<launch>
  <node pkg="bitathome_vision_module" name="face_detector" type="face_detector.py" output="screen">

    <remap from="input_rgb_image" to="/camera/rgb/image_color" />

   	<param name="cascade_1" value="$(find bitathome_vision_module)/data/haar_detectors/haarcascade_frontalface_alt2.xml" />
  	<param name="cascade_2" value="$(find bitathome_vision_module)/data/haar_detectors/haarcascade_frontalface_alt.xml" />
  	<param name="cascade_3" value="$(find bitathome_vision_module)/data/haar_detectors/haarcascade_profileface.xml" />

  </node>
</launch>
